exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -83,56 +84,77 @@ public String getContent() {
         maxRecords = Math.max(sp.getRecords(), maxRecords);
       }
 
-      builder.appendFormattedInteger(maxBatches, null);
-      builder.appendFormattedInteger(maxRecords, null);
-      builder.appendBytes(op.getPeakLocalMemoryAllocated(), null);
+      builder.appendFormattedInteger(maxBatches, null, null);
+      builder.appendFormattedInteger(maxRecords, null, null);
+      builder.appendBytes(op.getPeakLocalMemoryAllocated(), null, null);
     }
     return builder.build();
   }
 
-  public static final String[] OPERATORS_OVERVIEW_COLUMNS = {"Operator ID", "Type", "Min Setup Time", "Avg Setup Time",
-    "Max Setup Time", "Min Process Time", "Avg Process Time", "Max Process Time", "Min Wait Time", "Avg Wait Time",
-    "Max Wait Time", "Avg Peak Memory", "Max Peak Memory"};
+  public static final String[] OPERATORS_OVERVIEW_COLUMNS = {"Operator ID", "Type",
+    "Avg Setup Time", "Max Setup Time",
+    "Avg Process Time", "Max Process Time", "Min Wait Time", "Avg Wait Time", "Max Wait Time",
+    "% Fragment Time", "% Query Time", "Rows",
+    "Avg Peak Memory", "Max Peak Memory"};
 
-  public void addSummary(TableBuilder tb) {
+  public static final String[] OPERATORS_OVERVIEW_COLUMNS_TOOLTIP = {null, null,
+    null, null, null, null, null, null, null,
+    "% of time that the major fragment spent in operator", "% of Query time spent in operator", "Records emitted by the operator",
```

Comment:
Rows emitted would be correct. For all the operators, this is a tally of the outgoing rows and not the rows processed. 
Changed text for the rest.